cmd/list.go

```diff
@@ -177,6 +178,8 @@ func (cooked cookedListCmdArgs) processProperties(object StoredObject) string {
 			builder.WriteString(propertyStr + ": " + object.contentType + "; ")
 		case contentEncoding:
 			builder.WriteString(propertyStr + ": " + object.contentEncoding + "; ")
+		case contentMD5:
+			builder.WriteString(propertyStr + ": " + object.contentMD5 + "; ")
```

Comment:
Consider base64 encoding or using some standard way to display an MD5 to an end user

Reply:
Good idea